src/main/java/de/tum/in/www1/artemis/web/rest/dto/examevent/ExamLiveEventDTO.java

```diff
@@ -20,52 +17,6 @@
     @JsonSubTypes.Type(value = ProblemStatementUpdateEventDTO.class, name = "problemStatementUpdate"),
 })
 // @formatter:on
-public abstract class ExamLiveEventDTO {
-
-    private Long id;
-
-    private String createdBy;
-
-    private Instant createdDate;
-
-    public Long getId() {
-        return id;
-    }
-
-    public void setId(Long id) {
-        this.id = id;
-    }
-
-    public String getCreatedBy() {
-        return createdBy;
-    }
-
-    public void setCreatedBy(String createdBy) {
-        this.createdBy = createdBy;
-    }
-
-    public Instant getCreatedDate() {
-        return createdDate;
-    }
-
-    public void setCreatedDate(Instant createdDate) {
-        this.createdDate = createdDate;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-        ExamLiveEventDTO that = (ExamLiveEventDTO) o;
-        return Objects.equals(id, that.id) && Objects.equals(createdBy, that.createdBy) && Objects.equals(createdDate, that.createdDate);
-    }
+public interface ExamLiveEventDTO {
```

Comment:
since this does nothing, should we remove it?

Reply:
but why are there no fields like in the old class DTO? (id, createdBy, createdDate)?

Reply:
It is used in some cases as response type which makes sense to me,